mantidproject/mantid
# Added deprecation notice to ISIS Reflectometry (Old).

Adds deprecation notice to the ISIS Reflectometry (Old) interface.

**To test:**
1. Open the ISIS Reflectometry (Old) interface.
2. Check that the title ends with `- DEPRECATED`.
3. Check that the deprecation notice was printed as a warning in the results log.

Fixes #21103 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http:Refl//www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
